### PR TITLE
Use labels for loops in template-config to reduce amount of data in output

### PR DESCRIPTION
**Pull request by igloo777**
### Proposed changes

Current tasks including loop, do not use labels. Thus, at the output we can see all the data of the loops items.
This little PR adds labels to tasks inside `template-config.yml`. 
Some label defaults have been taken from tasks content.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/ansible-role-nginx-config/blob/main/CONTRIBUTING.md) document
- [x] I have added Molecule tests that prove my fix is effective or that my feature works
- [x] I have checked that any relevant Molecule tests pass after adding my changes
- [x] I have updated any relevant documentation (`defaults/main/*.yml`, `README.md` and `CHANGELOG.md`)

### CODE REVIEW

File: tasks/config/template-config.yml [hunk 1]
```diff
@@ -49,8 +49,9 @@
     state: directory
     owner: "{{ nginx_config_main_template.user | default('nginx') }}"
     mode: 0755
-  loop:
-    "{{ nginx_config_http_template }}"
+  loop: "{{ nginx_config_http_template | map(attribute='config') }}"
```
Comment (alessfg) [inline tasks/config/template-config.yml:52]: The extra attribute you added breaks the task logic
```suggestion
  loop: "{{ nginx_config_http_template }}"
```

**Review by alessfg — CHANGES_REQUESTED** [review]
Thanks for the great suggestions! Can you also add a loop label to the proxy cache directories task and update the Changelog?

Something like
```
loop_control:
   label: "{{ item.1.path | default('config.proxy.cache_path.path undefined') }}"
```
should do the trick 😄

File: tasks/config/template-config.yml (hunk 1 above)
Comment (igloo777) [reply tasks/config/template-config.yml:52]: Yes, you are right.

**Comment by igloo777** [comment]
> Thanks for the great suggestions! Can you also add a loop label to the `Ensure NGINX proxy cache directories exist` task and update the Changelog?
> 
> Something like
> 
> ```
> loop_control:
>    label: "{{ item.1.path | default('config.proxy.cache_path.path undefined') }}"
> ```
> 
> should do the trick smile

Done! 
Could you please review it?

**Review by alessfg — APPROVED** [review]

**Pull request merged**